UI/signin.html

```diff
@@ -36,34 +36,38 @@
             <!--end navbar-->
         </div>
         <section>
-            <div class="epic-shelter">
-                <div class="row">
-                    <div class="column"></div>
-                    <div class="column">
-                        <div class="form">
-                            <h1 class="text-center nav__logo app-color">Welcome Back</h1>
-                            <div class="form-card">
-                                <form>
-                                    <div class="form-row">
-                                        <label for="name">Email:</label>
-                                        <input type="email" class="form-control" placeholder="Enter your email" required minlength="3" maxlength="30">
-                                    </div>
+            <div class="shelter">
```

Comment:
Do you not believe this should be in another PR/branch?

Reply:
alright,i'll work on it